src/RigidBodyTreeInspector.jl

```diff
@@ -152,4 +153,20 @@ function inspect(mechanism; show_inertias::Bool=false, randomize_colors::Bool=tr
         end, [Interact.signal(w) for w in widgets]...)
 end
 
+import Base: one
+one(::Type{Array{Float64,1}}) = 1. # FIXME: this is a hack to get gridded interpolation from Interpolations to work on vectors
+
+function animate(vis::Visualizer, mechanism::Mechanism{Float64}, times::Vector{Float64}, configurations::Vector{Vector{Float64}};
+    fps::Float64 = 30., realtimerate::Float64 = 1.)
+    state = MechanismState(Float64, mechanism)
+    dt = 1. / fps
+    interpolated_configurations = interpolate((times,), configurations, Gridded(Linear()))
+    for t in times[1] : dt : times[end]
+        tic()
```

Comment:
probably better to use @elapsed instead of tic and toq, since any call to tic inside set_configuration or draw will screw up the global tic/toq timer.